native/python/include/jp_pythontypes.h

```diff
@@ -44,36 +55,33 @@ typedef _object PyObject;
  * state how we want this object reference to be treated.  Each
  * policy will produce different actions on the creation of
  * a reference wrapper.
+ *
  */
-namespace JPPyRef
+class JPPyObject
 {
+	/** Create a new reference to a Python object.
+	 *
+	 * @param obj is the python object.
+	 */
+	JPPyObject(PyObject* obj, int i);
```

Comment:
what is the i parameter used for? If you document the obj parameter, I'd recommend to do the same for i.

Reply:
The i parameter was placed simply to make sure that signature is not hit except by the static method callers.  I could just use explicit but I needed to make sure it busted in every place the only method was used.  I can remove the extra parameter now if you would like me to.